README.md

```diff
@@ -104,7 +104,7 @@ To configure on a tour declaration, use `ui-tour-<option-name>="optionValue"`
 | backdropZIndex       | number   | 10000                     | Z-index of the backdrop. Popups will be positioned relative to this.                                                                                                                                                                         |
 | backdropBorderRadius | number   | 0                         | If your target element has a border radius, add it here so that the backdrop conforms to it.                                                                                                                                                 |
 | backdropPadding      | string   | ""                       | Adds padding around the target element on the backdrop. Same format as CSS padding rule.                                                                                                                                                 |
-| templateUrl          | string   | "tour-step-template.html" | Used as the template for the contents of the popup (see Angular UI Tooltip docs).                                                                                                                                                            |
+| template             | string   | ""                        | A string of HTML for the contents of the popup |
```

Comment:
I can't accept any PRs with breaking changes right now. I would suggest _adding_ a `template` option and putting it into `$templateCache` so that the existing interface doesn't break.